mono/metadata/marshal.c

```diff
@@ -1708,27 +1706,8 @@ emit_object_to_ptr_conv (MonoMethodBuilder *mb, MonoType *type, MonoMarshalConv
 #endif /* DISABLE_COM */
 
 	case MONO_MARSHAL_CONV_SAFEHANDLE: {
-		int dar_release_slot, pos;
+		int pos;
 		
-		dar_release_slot = mono_mb_add_local (mb, &mono_defaults.boolean_class->byval_arg);
```

Comment:
mono_mb_add_local has side effects and I remember seen code that uses a literal number when loading/storing it instead of the index returned. That's why I asked.